packages/react-vapor/src/components/flatSelect/FlatSelectSelectors.ts

```diff
@@ -0,0 +1,16 @@
+import {createSelector} from 'reselect';
+import * as _ from 'underscore';
+
+import {IReactVaporState} from '../../ReactVapor';
+import {IFlatSelectState} from './FlatSelectReducers';
+
+const getInput = (state: IReactVaporState, props: {id: string}): IFlatSelectState => {
+    return _.findWhere(state.flatSelect, {id: props.id});

```

Comment:
add an initial value to avoid an undefined as return

`const initialFlatSelectState = {id: undefiend, selectedOptionId: undefiend};`